server/etc/e-smith/templates/etc/dovecot/quota.passwd/10base

```diff
@@ -16,5 +16,9 @@
         $quota = int(($quota << 10 ) / 10.0);
 
         $OUT .= sprintf("%s:::::::userdb_quota_rule=*:bytes=%dM\n", $_->key, $quota);
+
+        # due to bug dev/#5837 we need to set also the quota to the short name format
+        (my $shortName = $_->key) =~ s/(@.*)$//; # strip domain
```

Comment:
According to [perlvar](https://perldoc.perl.org/perlvar.html#SPECIAL-VARIABLES) a `$.` variable exist. Escaping the `@` character could be required here.

```suggestion
        (my $shortName = $_->key) =~ s/\@.*$//; # strip domain
```

Reply:
We have the same code running since some here, but I do not exclude we need to improve the perl syntax.

@stephdl could you please verify and then merge the patch?

Reply:
the two ways are valid but I went to the @DavidePrincipi ways, it is close of first mine